assets/javascripts/discourse/components/babble-composer.js.es6

```diff
@@ -1,6 +1,8 @@
 import { showSelector } from "discourse/lib/emoji/emoji-toolbar";
+import userSearch from "discourse/lib/user-search";
 
 export default Ember.Component.extend({
+  userSearch: userSearch,
```

Comment:
I have no idea why I have to do this, but 'showSelector' works fine. Javascript makes me mad sometimes.